Support page (#279)
trilitech/tezos-developer-docs: docs/overview/support.md

---
title: Support
last_update:
  date: 24 January 2024
---

> There are many places that you can use to reach out for developer support, including:

Reviewer: Let's not duplicate the list of resources, just point to the existing one:
```suggestion
If you don't find the needed answers in the documentation or need clarifications, there are many places that you can use to reach out for developer support, see <https://tezos.com/community/>.
```

> There are many places that you can use to reach out for developer support, including:

Reviewer: I see that this section still duplicates the whole list of interactive channels. Did you forget to push this change?

> There are many places that you can use to reach out for developer support, including:

Author: Removed the list of social media resources and just linked to the Community page.

- Tezos Discord: https://discord.gg/yXaPy6s5Nr
- Tezos sub-reddit: https://www.reddit.com/r/tezos/
- Tezos Stack Exchange: https://tezos.stackexchange.com
- Tezos Element community: https://riot.tzchat.org/
- #tezos channel on Freenode

For more social media links, see https://tezos.com/community/.

> 

Reviewer: It's a good idea to list the other documentation website. We can add more value by making it a documentation map, clearly defining the scope of each website, including the current one, to avoid the impression of useless duplication and create a clear mental model for developers:
```suggestion

The current website (https://docs.tezos.com/) is not exhaustive. It aims covering the main topics concerning Tezos developers and the main Tezos features, in the form of documentation and tutorials.
```

> 

Reviewer: I think it would be better to start with the other docs and mention interactive support only after. This could filter the use of interactive resources, which are scarce, especially if we scale up as planned.

Author: Yes, good idea, thanks for the additions.

Reviewer: I see that the Documentation section still comes after the Social media section. My suggestion was to swap them, so people at least see the list of all doc sites first before asking for support.

> Other Tezos developer documentation is available at these sites:

Reviewer: ```suggestion
More detailed documentation on Tezos in general and on specific tools used in Tezos is available at these sites:
```

> - https://opentezos.com/ for all developer-related concepts

Reviewer: ```suggestion
- https://opentezos.com/ is a full training course on Tezos concepts, mostly for developers but also for bakers and other users, in the form of in-depth tutorials with exercises
```

> - https://tezos.gitlab.io/ for the Octez suite

Reviewer: ```suggestion
- https://tezos.gitlab.io/ is the official documentation of the Octez suite, a complete implementation of the Tezos platform, including a node, a client, a baker, and other tools
```

> - https://taqueria.io/ for Taqueria

Reviewer: ```suggestion
- https://taqueria.io/docs/intro/ is the official documentation of [Taqueria](https://taqueria.io/), a flexible framework for building Web3 applications on Tezos 
```

> - https://taqueria.io/ for Taqueria

Reviewer: ```suggestion
- https://taqueria.io/docs/intro/ is the official documentation of [Taqueria](https://taqueria.io/),  a seamless development workflow to easily build, test and deploy your Tezos applications
```

> - https://tezostaquito.io/ for Taquito

Reviewer: ```suggestion
- https://tezostaquito.io/docs/quick_start is the official documentation of [Taquito](https://tezostaquito.io/), a TypeScript library for developing Tezos applications faster and easier
```

> - https://ligolang.org/ for LIGO

Reviewer: same kind of remark here and for the rest of the items

> - https://smartpy.io/ for SmartPy

Reviewer: ```suggestion
- https://smartpy.io/manual/introduction/overview is the official documentation of the [SmartPy](https://smartpy.io/) language for writing Tezos smart contracts
```

> - https://docs.etherlink.com/ for Etherlink

Reviewer: Care about including https://archetype-lang.org ?

Author: Added Archetype

> - https://docs.etherlink.com/ for Etherlink

Reviewer: ```suggestion
- https://docs.etherlink.com/ is the official documentation for [Etherlink](https://www.etherlink.com), an EVM-compatible layer-2 blockchain running on Tezos, implemented as a [smart rollup](../architecture/smart-rollups)
```
